lints/lint_dnsname_bad_character_in_label.go

```diff
@@ -1,50 +1,36 @@
 package lints
 
 import (
+	"regexp"
+
 	"github.com/zmap/zcrypto/x509"
 	"github.com/zmap/zlint/util"
-	"regexp"
-	"strings"
 )
 
 type DNSNameProperCharacters struct {
 	CompiledExpression *regexp.Regexp
 }
 
 func (l *DNSNameProperCharacters) Initialize() error {
-	const dnsLabelRegex = "^[A-Za-z0-9*_-]+$"
-	l.CompiledExpression = regexp.MustCompile(dnsLabelRegex)
-	return nil
+	const dnsNameRegexp = `^(\*\.)?(\?\.)?(A-Za-z0-9*_-]+\.)*[A-Za-z0-9*_-]*$`
```

Comment:
There can be more than one "redacted" label: https://crt.sh/?id=250738174

Reply:
I will update the regex. 
You are aware though that the master branch code doesn't handle this case either?
I was refactoring the code without introducing any functional changes.

Reply:
Isn't it handled by `util.RemovePrependedQuestionMarks`?

Reply:
```
func RemovePrependedQuestionMarks(domain string) string {
	for strings.HasPrefix(domain, "?.") {
		domain = domain[2:]
	}
	return domain
}
```
Doesn't it remove only the first occurrence of `?.` ?

Reply:
Fixing now.